### PR TITLE
Breaking changes for the Generac MobileLink Binding

### PR DESCRIPTION
**Pull request by digitaldan**
See https://github.com/openhab/openhab-addons/pull/14638

### CODE REVIEW

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -105,6 +105,7 @@ ALERT;JavaScript Scripting Automation: ItemHistory: historicState, maximumBetwee
 ALERT;KNX Binding: Units of Measurements have been introduced, make sure you use correct units in the state description. Rules may need to be adapted. Several items changed from PercentType to QuantityType, i.e. raw values increase by a factor of 100.
 ALERT;Netatmo Binding: Due to API authorization process change scheduled on the 2023/04/17 - refresh Token is no more stored in thing configuration, thus can be removed from things config files. If it remains, it'll be ignored.
 ALERT;Netatmo Binding: New scope has been added for the introduction of the Carbon Monoxide Alarm. Authorization process has to be replayed (and former refreshToken can be removed from things config files -- see above).
+ALERT;Generac MobileLink Binding: Due to an API change, existing Generator Things will need to be deleted and added again. Additionally, existing Items will need to be adjusted to reflect the updated Generator Thing channels.
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:108]: Can you move this new `ALERT` before `ALERT;Hue emulation` so the list remains sorted?

Comment (digitaldan) [reply distributions/openhab/src/main/resources/bin/update.lst:108]: Will do!

**Comment by digitaldan** [comment]
https://github.com/openhab/openhab-addons/pull/14638 has been merged, thanks!

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**